### DISCUSSION TITLE
wght avar

### DISCUSSION THREAD

**Discussion by chrissimpkins** [googlefonts/googlesans#278] [General]
_Re-posting from a different repository so that this is available to all design teams_

---

I took a look at the 380, 555, 734 wght avar and our master definitions.  This does not define a linear relationship with simple extension of internal values to more extreme points on the wght axis.

Here is where we are (solid line with avar, dashed line is a linear relationship between user and internal):

![avar-wght](https://user-images.githubusercontent.com/4249591/109558406-5e96cb80-7aa7-11eb-9c8e-1c5a2efbd1a9.png)

C/W a linear relationship, there is a more rapid increase in weight from user 400 - 500, interpolated 500 is heavier, and there is a less rapid increase in weight from user 500 - 700.  

The slope of the line is (734 - 380) / (700 - 400) = 1.18.

That would put us at a user=500 internal value of 380 + 1.18(100) = 498 to maintain linearity.  We are at user 500 = internal 555

**Comment by chrissimpkins** [comment]
cc @justvanrossum @JeremieHornus

**Reply by justvanrossum** [reply]
Thanks!

**Reply by JeremieHornus** [reply]
Thank you! Noted

**Comment by chrissimpkins** [comment]
cc @tphinney

**Comment by chrissimpkins** [comment]
cc @MariannaPaszkowska @dberlow @sannorozco This is the weight avar definition in Google Sans.  Let's not add a wght avar in GS Flex until we hold the discussion about how to handle an avar definition (if any) across the entire axis range.  I'd like to involve @davelab6 in this conversation because he has argued for avar1 on Google Sans Flex wght axis in the past.
